requirements.txt

```diff
@@ -2,4 +2,5 @@ gwyfile>=0.2.0
 numpy<2.0.0
 setuptools
 scipy
-gsffile
\ No newline at end of file
+gsffile
+pandas
```

Comment:
Pandas should have a version requirement. But keep it as open as possible. For example, Orange has `pandas>=1.4.0`.